.storybook/main.ts

```diff
@@ -19,8 +19,6 @@ const config: StorybookConfig = {
     ...config,
     VITE_STORYBOOK_PUBLISHABLE_API_KEY:
       process.env.VITE_STORYBOOK_PUBLISHABLE_API_KEY || "",
-    VITE_STORYBOOK_RESTRICTED_SECRET:
-      process.env.VITE_STORYBOOK_RESTRICTED_SECRET || "",
```

Comment:
This env var is no longer needed. With the new flow, we don't need a payment_intent or setup_intent to render the payment form 😄